Source/WebCore/rendering/HitTestResult.cpp

```diff
@@ -528,6 +529,48 @@ void HitTestResult::enterFullscreenForVideo() const
 #endif
 }
 
+bool HitTestResult::videoHasSubtitles() const
+{
+#if ENABLE(VIDEO)
+    if (HTMLMediaElement* mediaElement = this->mediaElement()) {
```

Comment:
Nit: this can be `auto* mediaElement ...`

An <audio> element won't display subtitles, so I think this should also check to make sure this is a <video> element.